src/client/components/fields/OwnedRelationsManager.tsx

```diff
@@ -69,13 +73,27 @@ const OwnedRelationsManager: React.FC<OwnedRelationsManagerProps> = ({
     setSelected((prev) => prev.filter((_, i) => i !== idx));
   };
 
+  const listableRels: FieldsAndRels = manifest.relations.filter(
```

Comment:
Please extract these const... it seems that you are using it in like 3 other files 👀

Reply:
Good note!